core/src/banking_stage.rs

```diff
@@ -729,11 +750,19 @@ impl BankingStage {
             Self::push_unprocessed(&mut unprocessed_packets, msgs, unprocessed_indexes);
 
             if processed < verified_txs_len {
+                let poh = poh.lock().unwrap();
```

Comment:
Does this hold the poh lock for the duration of this call intentionally ?

Reply:
Good point. It should not be required.

Reply:
the latest patch should fix it